web_digital_sign/users.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
```

Comment:
Put it under 'models' folder and call it res_users.py for following conventions.